core-web/libs/portlets/dot-analytics/portlet/src/lib/dot-analytics-dashboard/constants/dot-analytics.constants.ts

```diff
@@ -0,0 +1,52 @@
+import { TimeRange } from '@dotcms/portlets/dot-analytics/data-access';
+
+import { FilterOption } from '../types';
+
+/** Available time period options for analytics data filtering */
+export const TIME_PERIOD_OPTIONS: FilterOption[] = [
+    { label: 'analytics.filters.time-period.1week', value: 'from 7 days ago to now' },
+    { label: 'analytics.filters.time-period.1month', value: 'from 30 days ago to now' }
+];
+
+/** Default time period selection (1 week) */
+export const DEFAULT_TIME_PERIOD: TimeRange = 'from 7 days ago to now';
```

Comment:
Oh, you have the constant here, check if you can reuse it or go for the enum approach